src/libs/utils/consoleprocess.cpp

```diff
@@ -364,7 +368,7 @@ void ConsoleProcess::setAbortOnMetaChars(bool abort)
 
 qint64 ConsoleProcess::applicationMainThreadID() const
 {
-    if (HostOsInfo::isWindowsHost())
+    if (HostOsInfo::isWindowsHost() || HostOsInfo::isOs2Host())
```

Comment:
```suggestion
    if (HostOsInfo::isDosLikeHost())
```
This requires a suggestion listed below.